sources/packages/backend/apps/queue-consumers/src/processors/assessment/cancel-application-assessment.processor.ts

```diff
@@ -0,0 +1,83 @@
+import { Process, Processor } from "@nestjs/bull";
+import { Job } from "bull";
+import { CancelAssessmentQueueInDTO } from "@sims/services/queue";
+import {
+  DisbursementScheduleService,
+  WorkflowClientService,
+} from "@sims/services";
+import { QueueNames } from "@sims/utilities";
+import { StudentAssessmentService } from "../../services";
+import { DataSource } from "typeorm";
+import { ZeebeGRPCError, ZeebeGRPCErrorTypes } from "@sims/services/zeebe";
+
+/**
+ * Process the workflow cancellation.
+ */
+@Processor(QueueNames.CancelApplicationAssessment)
+export class CancelApplicationAssessmentProcessor {
+  constructor(
+    private readonly dataSource: DataSource,
+    private readonly workflowClientService: WorkflowClientService,
+    private readonly studentAssessmentService: StudentAssessmentService,
+    private readonly disbursementScheduleService: DisbursementScheduleService,
+  ) {}
+
+  /**
+   * Cancel the workflow instance and rollback overawards, if any.
+   * @param job information to perform the process.
+   * @returns log message.
+   */
+  @Process()
+  async cancelAssessment(
+    job: Job<CancelAssessmentQueueInDTO>,
+  ): Promise<string> {
+    await job.log(`Start processing, attempt ${job.attemptsMade + 1}.`);
```

Comment:
so, we are going this job. logs right? no more nest logger?

Reply:
I am not sure. I added to this PR to start the discussion here.
I like the idea of using it because it is easy to see in the Bull Dashboard.
The main difference is the logs that we send to the console are also kept in Openshift and can be accessed by Kibana, as far as I know.

Reply:
I have a doubt here. Job.log() is great, but by just doing job.log() aren't we missing to log to the console which will be watched be tools like kibana as you mentioned. May be at least error should be logged in console?

Reply:
@ann-aot @dheepak-aot I will use the Nest logger, for now, maybe we can explore `job.log` for a long-running process like Fed Restrictions where we can even report the progress also. I
For the logging errors, please notice that the unknown error is still thrown and the `job.log` for the `NOT_FOUND` is not considered an error.